igvm/templates/user_data.cfg

```diff
@@ -0,0 +1,92 @@
+        TLpK0wAT0n9RHQOQaeBfZ/OPYrZbKPsDXo/094J6Z9UffZMGPaEgt9+hR5Z62//T
+        EyEUHNoFukDBBJcy8c+GnPpmTQ17XzhxKI0pPJ2H5aDNeh5b3D6y0WXgpiOKiNxr
+        0wX1t0pyDO1Y7JE/51qq7ETol0ezws7vVLrxFi0C7s2mjNWivS5J3PcvF+TK+a1C
+        wYG0XDf3aV2OnxhDWz9YVtyY8NLEmLzsbmCigzXy+1h23TBXaVlKXaw9k8dxnKM9
+        UStc3hMIN50aqK0Tucbz2yj9+QQf9YHfzWOfqP9m0nm2CCmx1w+GZE9MLAhszNrl
+        TW7UN+hEyA1Z
+        =mJCR
+        -----END PGP PUBLIC KEY BLOCK-----
+    basestretch_stable.list:
+      source: "deb http://update-int.ig.local/ basestretch stable"
+      key: |
+        -----BEGIN PGP PUBLIC KEY BLOCK-----
+        Version: GnuPG v1
+
+        mQINBFjsyv0BEADlyvUwquhi/PzwaYIG2JMNHEMnUc3jKaHd3SAW6kRk6uzao9B1
+        8fBMRnuGFtfrmRxa7CvqTTcn3hXl8h7kZQICBHXJrNXB2KDs4gnNDXbjXBuLJyW5
+        jVe8Ghn7GEJbzUAu+iNPBKi1UYkUoMSk+sAbXqOaUJQDJSaOU4A7C9wL1mAxsWNh
+        ucpCBtt9HTTnVWiZiLAtvop7LJEgXEpQbht/jW7mdMX5yAbWfYUJMdXFMCADz5r6
+        mbtuun5BaDae31dgA//IPI+Im8PxJBgdyyETfhoSbThiFDk4K1UgXXlAwYxR7TVx
+        PjvWN8HDlbBl4TvM29ppz8bZJARQx3h1SYIArYJ2Xc9lQwsbK0KzL/w1NmZsiPzt
+        bpM0Fs0Q7+oQScxuIolpu4c77siEged2R6dnXR/er+zN5/GgTdaIEcQnTRI9F8bZ
+        KrhIy/TSLgRnxZYFyNE30yQtJD2WR9fuh6zT72LY0rvqQcHw7Ze3qW2ZgORswdGW
+        masHMJ/LkXv75MiV4KMBwweYp6OW9ewisdrZEijQS0NEcZ+YFC288dLJeWLpPkzd
+        czTO/v/bLDQ/Uk8aOcldtJfB4BjBW9v7JjvLcKtCMBsW5P4o3IjVJt+08FfGs7cy
+        aHJifH6RCM2GBDUaNAoPPZmioiKZcw6twMhdrjzVHiPFmolEXzItLmFHNwARAQAB
+        tDphZG1pbiAoSW5ub2dhbWVzIGRlYmlhbiByZXBvc2l0b3J5IGtleSkgPGl0QGlu
+        bm9nYW1lcy5jb20+iQI3BBMBCgAhBQJY7Mr9AhsDBQsJCAcDBRUKCQgLBRYCAwEA
+        Ah4BAheAAAoJEKy7fO0+1wCYz5oP/iBSLv3h+DRCAMeV68EquhyOvGUIZuZ910wL
+        7wW9lqoUmKCdrSoOvXxpE7z1TFegRfUtbnXYJYmELwp0SD5F/60ScCoW7bbTQdK4
+        FHms0CYjUcJnRjRrQVyE0fiChql6CYif2tWWOR7L+COBeIKrSaqXDPw2X5A0mJ+Q
+        FRcVhNbz02FIfr1mIECVshN34x5RMnmJtd6ZNhcbldS26FhxC5SrzFkAPExCFBVL
+        iAsfEK51ED4QOfnRRYOlbBvDwL3XQBZtFVtznwI1DLg/JNVVXFM8sDei+eL3O5yo
+        OVEuTBRvNH7ONlFRcBdFCdfSi1VvKVY2JEQX4R4BHyZvD/B2pcBeTM5puav5QuwK
+        XppwHg4dBbs1BrRsHWGMdqTJWGkwwVUdt3af6WSWD6BA+rv2I3DM7BMt6t/FK1BM
+        apHGY2fj3gK8pWyi13R8tryr4ZdIa/RnwxWt8ySWclgDVSPxTcNE1Zn2fYDB/Ykv
+        TyF52dn3+6UW6mnjL45Hictqo9A6aOxre5dlwEyHXhurs/7UP0Ell20UEN1VsKDP
+        E2V8pq+JFXQ+F2sLMEar4L5teopk78L7t0FibqmKcFRVgvseBbjm0xHfdTk4mXCD
+        wbaBtX+E9qhd5e4brEYNx9I9I2fvXGw5sX6Qdty9HnEQOKGMshEAfl2iCzJ56n4C
+        AkPnU50v
+        =KPbA
+        -----END PGP PUBLIC KEY BLOCK-----
+
+package_update: true
+package_upgrade: true
+packages:
+ - puppet-agent
+ - puppet-msgpack
+
+runcmd:
+  - echo "# We need to use external IPs for puppet here" >> /etc/hosts
+  - echo 212.53.146.132 {{ puppet_ca }} >> /etc/hosts
+  - echo 212.53.146.131 {{ puppet_master }} >> /etc/hosts
+  - /opt/puppetlabs/puppet/bin/puppet agent --detailed-exitcodes --fqdn={{ fqdn }} --server={{ puppet_master }} --ca_server={{ puppet_ca }} --no-report --waitforcert=60 --onetime --no-daemonize --verbose
```

Comment:
Can you make it fit in 80 columns?

Reply:
Looks ugly, bit now it's fitting 80 columns.